code/controllers/subsystem/polling.dm

```diff
@@ -0,0 +1,312 @@
+		// Opt-out for admins whom are currently adminned.
+		if((!candidate_mob.client.prefs.read_preference(/datum/preference/toggle/ghost_roles_as_admin)) && candidate_mob.client.holder)
+			continue
+		if(!is_eligible(candidate_mob, role, check_jobban, ignore_category))
+			continue
+
+		SEND_SOUND(candidate_mob, 'sound/misc/notice2.ogg')
+		if(flash_window)
+			window_flash(candidate_mob.client)
+
+		// If we somehow send two polls for the same mob type, but with a duration on the second one shorter than the time left on the first one,
+		// we need to keep the first one's timeout rather than use the shorter one
+		var/atom/movable/screen/alert/poll_alert/current_alert = LAZYACCESS(candidate_mob.alerts, category)
+		var/alert_time = poll_time
+		var/alert_poll = P
+		if(current_alert && current_alert.timeout > (world.time + poll_time - world.tick_lag))
+			alert_time = current_alert.timeout - world.time + world.tick_lag
+			alert_poll = current_alert.poll
+
+		// Send them an on-screen alert
+		var/atom/movable/screen/alert/poll_alert/A = candidate_mob.throw_alert(category, /atom/movable/screen/alert/poll_alert, timeout_override = alert_time, no_anim = TRUE)
+		if(!A)
+			continue
+
+		A.icon = ui_style2icon(candidate_mob.client?.prefs?.read_preference(/datum/preference/choiced/ui_style))
+		A.desc = "[question]"
+		A.show_time_left = TRUE
+		A.poll = alert_poll
+		A.poll.alert_button = A
+
+		// Sign up inheritance and stacking
+		var/inherited_sign_up = FALSE
+		for(var/existing_poll in currently_polling)
+			var/datum/candidate_poll/P2 = existing_poll
+			if(P != P2 && P.hash == P2.hash)
+				// If there's already a poll for an identical mob type ongoing and the client is signed up for it, sign them up for this one
+				if(!inherited_sign_up && (candidate_mob in P2.signed_up) && P.sign_up(candidate_mob, TRUE))
+					A.update_signed_up_alert()
+					inherited_sign_up = TRUE
+
+		// Image to display
+		var/image/I
+		if(pic_source)
+			if(!ispath(pic_source))
+				var/atom/PS = pic_source
+				var/old_layer = PS.layer
+				var/old_plane = PS.plane
+				PS.plane = A.plane
+				PS.layer = FLOAT_LAYER
+				A.add_overlay(PS)
+				PS.layer = old_layer
+				PS.plane = old_plane
+			else
+				I = image(pic_source, layer = FLOAT_LAYER)
+		else
+			// Just use a generic image
+			I = image('icons/effects/effects.dmi', icon_state = "static", layer = FLOAT_LAYER)
+
+		if(I)
+			I.plane = A.plane
+			A.add_overlay(I)
+
+		// Chat message
+		var/act_jump = ""
+		if(isatom(pic_source) && isobserver(candidate_mob))
+			act_jump = "<a href='?src=[REF(A)];jump=1'>\[Teleport]</a>"
+		var/act_signup = "<a href='?src=[REF(A)];signup=1'>\[Sign Up]</a>"
+		var/act_never = ""
+		if(ignore_category)
+			act_never = "<a href='?src=[REF(A)];never=1'>\[Never For This Round]</a>"
+		to_chat(candidate_mob, "<big>[span_boldnotice("Now looking for candidates [role_name_text ? "to play as \an [role_name_text]." : "\"[question]\""] [act_jump] [act_signup] [act_never]")]</big>")
```

Comment:
```suggestion
		to_chat(candidate_mob, span_boldnotice("<big>Now looking for candidates [role_name_text ? "to play as \an [role_name_text]." : "\"[question]\""] [act_jump] [act_signup] [act_never]</big>"))
```

ideally we should have a `span_big()` or something but for the meantime this will do the exact same thing while having less wack strings